.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
+
+on: push
```

Comment:
I would include the changes you introduced in the master to not test the master and tags